src/frontend/apps/impress/src/features/left-panel/components/LeftPanelFavorites.tsx

```diff
@@ -53,23 +50,36 @@ export const LeftPanelFavorites = () => {
         >
           {invitations.map((doc) => (
             <Box
+              $direction="row"
+              $align="center"
+              $justify="space-between"
               $css={css`
                 padding: ${spacing['2xs']};
                 border-radius: 4px;
+                .pinned-actions {
+                  opacity: 0;
+                }
                 &:hover {
                   cursor: pointer;
+
                   background-color: var(--c--theme--colors--greyscale-100);
+                  .pinned-actions {
+                    opacity: 1;
+                  }
                 }
               `}
               key={doc.id}
             >
               <StyledLink href={`/docs/${doc.id}`}>
                 <SimpleDocItem showAccesses doc={doc} />
               </StyledLink>
+              <div className="pinned-actions">
```

Comment:
Is this effect just inside `LeftPanelFavorites` ? You don't want the same effect in DocsGridItem as well?